README.md

```diff
@@ -40,21 +40,20 @@ the build guide and set up all required paths and repositories accordingly.
 ### Quick start packages, Release 1.4.1
 
 The quick start packages (https://github.com/G-Node/nix-mx/releases)
-are compiled under macOS Sierra using Matlab 2016b and contain the compiled mex files, 
+are compiled under macOS Sierra using Matlab 2020a and contain a statically built mex files, 
 nix m-files, tests, and a `startup.m` script. To use nix-mx unzip the file and run the 
 `startup.m` script in MATLAB. This simply adds the current folder containing the mex 
 files to the MATLAB path. Do not change the file/folder structure.
 
-In order to have it working the respective NIX C++ library must be installed on the 
-system. The easiest way is using homebrew `brew install nixio`
-
 Once this is done, you may want to test it. Go to the MATLAB command line, change into 
 the nix-mx folder and execute:
 
 `>>RunTests`
 
 This will execute a bunch of tests, there may be warnings but no test should fail.
 
+*Note:* Apple might block the execution because they cannot verify the app. You can allow the execution in you SystemPreferences -> Security -> General 
```

Comment:
```suggestion
*Note:* Apple might block the execution because they cannot verify the app. You can allow the execution in your SystemPreferences -> Security -> General 
```